components/Player/PlayerActions.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { CardActions } from 'material-ui/Card';
+import RaisedButton from 'material-ui/RaisedButton';
+import ActionUpdate from 'material-ui/svg-icons/action/update';
+import ReactTooltip from 'react-tooltip';
+
+export default () => (
+  <CardActions style={{ width: '30%', boxSizing: 'border-box', padding: '15px 30px', textAlign: 'right' }}>
+    <div style={{ display: 'inline-block' }}>
+      <div data-tip data-for="update">
+        <RaisedButton
+          disabled={true}
+          icon={<ActionUpdate />}
+          style={{ minWidth: 50 }}
+        />
+      </div>
+      <ReactTooltip id="update" place="left" type="light" effect="float">
+        <div style={{ textAlign: 'left' }}>
+          Refresh Match History:
```

Comment:
Can we make this one continuous string?  Eventually we'll move them all out to language files and it's easier to deal with it that way.